Tasks/Gradle/testlib/tasklib-wrapper.ts

```diff
@@ -0,0 +1,158 @@
+// Redirects calls to vsts-task-lib to testlib if in a test environment.
+import tasklib = require('vsts-task-lib/task');
+import testlib = require('./testlib-main');
+
+// TODO: should go away when task lib
```

Comment:
Some parts of this code were copied from vsts-task-lib in order to retain function, and comments and TODOs were kept for continuity.